shunting-yard-replicator/src/test/java/com/hotels/shunting/yard/replicator/exec/external/MarshallerTest.java

```diff
@@ -21,20 +21,32 @@
 import java.nio.charset.StandardCharsets;
 import java.util.Arrays;
 
+import org.junit.Before;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.TemporaryFolder;
 
 import com.google.common.io.Files;
 
 import com.hotels.bdp.circustrain.api.conf.ReplicationMode;
+import com.hotels.shunting.yard.replicator.exec.conf.Graphite;
 
 public class MarshallerTest {
 
   public @Rule TemporaryFolder tmp = new TemporaryFolder();
 
   private final Marshaller marshaller = new Marshaller();
 
+  Graphite graphiteConfig = new Graphite();
+
+  @Before
+  public void init() {
```

Comment:
Can we not just do this in the one test below that uses graphite?

Reply:
There are 2/3 tests that use `graphiteConfig`: `typicalWithGraphite` and `generatePartitionFilter`